content/en/docs/Configuration/namespace-management.md

```diff
@@ -66,40 +77,88 @@ feature to Kiali. However, explicit naming of accessible namespaces can benefit
+spec:
+  api:
+    namespaces:
+      label_selector_include: kiali-enabled=true
+  deployment:
+    accessible_namespaces: ["**"]
 ```
 
-For further information on how this `api.namespaces.label_selector` interacts with `spec.deployment.accessible_namespaces` read the [Kiali CR Reference documentation](/docs/configuration/kialis.kiali.io/#.spec.api.namespaces.label_selector).
-
-To label a namespace you can use the following command. For more information see the [Kubernete's official documentation](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels).
+The example below hides all namespaces that have a label `infrastructure: system`:
 
+```yaml
+spec:
+  api:
+    namespaces:
+      label_selector_exclude: infrastructure=system
 ```
-  kubectl label namespace my-namespace kiali-enabled=true
-```
 
-Note that when deploying multiple control planes in the same cluster, you will want to set the label selector's value unique to each control plane. This allows each Kiali instance to select only the namespaces relevant to each control plane. Because in this "soft-multitenancy" mode `spec.deployment.accessible_namespaces` is typically set to an explicit set of namespaces (i.e. not `["**"]`), you do not have to do anything with this `label_selector`. This is because the default value of `label_selector` is `kiali.io/member-of: <spec.istio_namespace>` when `spec.deployment.accessible_namespaces` is not set to the "all namespaces" value `["**"]`. This allows you to have multiple control planes in the same cluster, with each control plane having its own Kiali instance. If you set your own Kiali instance name in the Kiali CR (i.e. you set `spec.deployment.instance_name` to something other than `kiali`), then the default label will be `kiali.io/<spec.deployment.instance_name>.member-of: <spec.istio_namespace>`.
+### Important Note About `label_selector_include`
+
+It is very important to understand how the `api.namespaces.label_selector_include` setting is used when `deployment.accessible_namespaces` is set to an explicit list of namespaces versus when it is set to `["**"]`.
+
+When `accessible_namespaces` is set to `["**"]` then `label_selector_include` simply provides a filter just like `api.namespaces.include`. The difference is `label_selector_include` filters by namespace label whereas `include` filters by namespace name. Thus, `label_selector_include` merely provides an additional way to limit what namespaces the Kiali user will see when `accessible_namespaces` is set to `["**"]`.
+
+When `accessible_namespaces` contains an explicit list of namespaces (i.e. is not set to `["**"]`), `label_selector_include` defines the label (name and value) that will be added by the operator to each namespace defined in `accessible_namespaces`. After the operator installs Kiali, every accessible namespace will have this label. Thus, Kiali can then use `label_selector_include` to select all namespaces as defined in `accessible_namespaces`.
```

Comment:
```suggestion
It is recommended to leave `label_selector_include` unset when `accessible_namespaces` is not set to `["**"]` (i.e. `accessible_namespaces` is set to an explicit list of namespaces).  This is because the operator adds the configured label (name and value) to each namespace defined by `accessible_namespaces`.  This allows the Kiali code to use the `label_selector_include` value to easily select all of the `accessible_namespaces`.  Unless you have a good reason to customize, the default value is recommended in this scenario.
```

Reply:
done